[2025-01-26T10:01:18.115+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-26T10:01:18.475+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_and_store_ebooks.fetch_ebook_data scheduled__2020-02-25T00:00:00+00:00 [queued]>
[2025-01-26T10:01:18.600+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_and_store_ebooks.fetch_ebook_data scheduled__2020-02-25T00:00:00+00:00 [queued]>
[2025-01-26T10:01:18.623+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-26T10:01:19.403+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_ebook_data> on 2020-02-25 00:00:00+00:00
[2025-01-26T10:01:19.597+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=5585) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-26T10:01:19.811+0000] {standard_task_runner.py:72} INFO - Started process 5625 to run task
[2025-01-26T10:01:19.654+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fetch_and_store_ebooks', 'fetch_ebook_data', 'scheduled__2020-02-25T00:00:00+00:00', '--job-id', '1292', '--raw', '--subdir', 'DAGS_FOLDER/app.py', '--cfg-path', '/tmp/tmp_8tc9paa']
[2025-01-26T10:01:20.015+0000] {standard_task_runner.py:105} INFO - Job 1292: Subtask fetch_ebook_data
[2025-01-26T10:01:21.662+0000] {task_command.py:467} INFO - Running <TaskInstance: fetch_and_store_ebooks.fetch_ebook_data scheduled__2020-02-25T00:00:00+00:00 [running]> on host 34edd304782d
[2025-01-26T10:01:24.491+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_and_store_ebooks' AIRFLOW_CTX_TASK_ID='fetch_ebook_data' AIRFLOW_CTX_EXECUTION_DATE='2020-02-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2020-02-25T00:00:00+00:00'
[2025-01-26T10:01:24.605+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-26T10:01:24.720+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-26T10:01:24.741+0000] {logging_mixin.py:190} INFO - Current task name:fetch_ebook_data state:running start_date:2025-01-26 10:01:18.489379+00:00
[2025-01-26T10:01:24.789+0000] {logging_mixin.py:190} INFO - Dag name:fetch_and_store_ebooks and current dag run status:running
[2025-01-26T10:01:24.795+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-26T10:01:25.312+0000] {job.py:229} INFO - Heartbeat recovered after 11.38 seconds
[2025-01-26T10:01:43.239+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-26T10:01:43.450+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-26T10:01:43.497+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fetch_and_store_ebooks, task_id=fetch_ebook_data, run_id=scheduled__2020-02-25T00:00:00+00:00, execution_date=20200225T000000, start_date=20250126T100118, end_date=20250126T100143
[2025-01-26T10:01:44.179+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-26T10:01:44.194+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-26T10:01:44.213+0000] {logging_mixin.py:190} INFO - Dag name:fetch_and_store_ebooks queued_at:2025-01-26 10:00:32.609013+00:00
[2025-01-26T10:01:44.219+0000] {logging_mixin.py:190} INFO - Task hostname:34edd304782d operator:PythonOperator
[2025-01-26T10:01:44.449+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-26T10:01:45.029+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-01-26T10:01:44.985+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-26T10:01:45.179+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
